### PR TITLE
Feature/40 text component UI

### LINKED ISSUE #40: Connect text component on front and back
DoD:
- [ ] on front are good inputs
- [ ] REST is sending correctly
- [ ] on front are correct results images

### PR DESCRIPTION
**Pull request by Derstilon**
image
Closes #40

### CODE REVIEW

File: ui/src/lib/components/Form.svelte [hunk 1]
```diff
@@ -69,39 +107,88 @@
...
                         on:change={changeDate}
                     />
                 </label>
             </div>
         {/if}
     </div>
+    <div class="inputContainer">
+        <p class="inputContainerTitle" on:click={handleClick("text")}>
+            <input type="checkbox" bind:checked={formData["text"].active} on:click|stopPropagation/>
+            Tetadata <i class="arrow {arrowDirection}" />
```
Comment (retkiewi) [inline ui/src/lib/components/Form.svelte:146]: The f**k is Tetadata

File: ui/src/lib/components/Form.svelte [hunk 2]
```diff
@@ -69,39 +107,88 @@
...
+        {#if moduleUis["text"].visible}
+            <div class="moduleForm" transition:slide>
+                <label
+                    >Has Text
+                    <input
+                        type="checkbox"
+                        bind:checked={formData["text"].hasText}
+                    />
+                </label>
+                {#if formData["text"].hasText}
```
Comment (retkiewi) [inline ui/src/lib/components/Form.svelte:157]: make the fields render as disabled instead of hiding them

**Review by retkiewi — CHANGES_REQUESTED** [review]

File: ui/src/lib/components/Form.svelte [hunk 3]
```diff
@@ -69,39 +107,88 @@
...
+                <label
```
[the remark this answers is not in the document]
Comment (Derstilon) [reply ui/src/lib/components/Form.svelte:158]: image

[the remark this answers is not in the document]
Comment (retkiewi) [reply ui/src/lib/components/Form.svelte:157]: niech będzie xd

**Review by retkiewi — APPROVED** [review]

**Pull request merged**